### DISCUSSION TITLE
Authorization

### DISCUSSION THREAD

**Discussion by PraveenRepswal** [community/community#81386] [New to GitHub]
Recently I authorized an application to access my GitHub account.
I just want to ask that it the Kharagpur Winter of Code have only access to my public data or can it control my GitHub account using my whole data?
![git](https://github.com/community/community/assets/129259520/c1bb519a-a020-44e5-906a-5ec417bb60ec)

**Comment by jzelAdmin2006** [comment]
Hi Praveen

I hope I will be able to help, even though the answer is really late now. If so, please **mark this as the answer**, I would be infinitely grateful!

Idk why nobody has answered this simple question for you. I have been working with GitHub Apps (Plugins) lately, and although I can't see all the scope definitions on your screenshot, it seems like this app doesn't have any writing access to your account. If so, you would be asked explicitly. The developers of the app also can't just change their rights in order to access your account, that would just result in a prompt where you would have to confirm the updated access rights.
To check the app's rights in retrospect, you can either check https://github.com/settings/installations or https://github.com/settings/apps/authorizations.
Here's an example of an app that has writing access to your account if you install it.

![Snag_661409c](https://github.com/community/community/assets/112761658/11d28d13-3e48-4b83-bc91-7deb6ecb2446)
![image](https://github.com/community/community/assets/112761658/473d0459-c05d-4818-90ef-b0ea99f3e089)

All the best
JZEL
